ERROR - 2022-09-05 16:50:36 --> Error connecting to the database: Unknown database 'delibery'
CRITICAL - 2022-09-05 16:50:36 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'delibery'
#0 C:\Users\diego\OneDrive\Documentos\Pre-especializacion\Proyecto Delivery\backend\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Users\diego\OneDrive\Documentos\Pre-especializacion\Proyecto Delivery\backend\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\Users\diego\OneDrive\Documentos\Pre-especializacion\Proyecto Delivery\backend\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Users\diego\OneDrive\Documentos\Pre-especializacion\Proyecto Delivery\backend\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\Users\diego\OneDrive\Documentos\Pre-especializacion\Proyecto Delivery\backend\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\Users\diego\OneDrive\Documentos\Pre-especializacion\Proyecto Delivery\backend\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\Users\diego\OneDrive\Documentos\Pre-especializacion\Proyecto Delivery\backend\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\Users\diego\OneDrive\Documentos\Pre-especializacion\Proyecto Delivery\backend\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\Users\diego\OneDrive\Documentos\Pre-especializacion\Proyecto Delivery\backend\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\Users\diego\OneDrive\Documentos\Pre-especializacion\Proyecto Delivery\backend\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\diego\\...')
#10 {main}
